salt/minion.py

```diff
@@ -1685,6 +1685,15 @@ def _fire_master_minion_start(self):
             tagify([self.opts['id'], 'start'], 'minion'),
         )
 
+    def grains_refresh_manual(self):
+        '''
+        Perform a manual grains refresh
+        '''
+        self.opts['grains'] = salt.loader.grains(
+            self.opts,
+            force_refresh=True,
+            proxy=salt.utils.is_proxy())
```

Comment:
`proxy` here is expecting a salt.loader.proxy reference but salt.utils.is_proxy() returns True or False.